### PR TITLE
feat: launch everything from the same docker compose

### PR DESCRIPTION
**Pull request by Rignchen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
-.idea
\ No newline at end of file
+.idea
+# vim backup files
+*.swp
```
Comment (jpilet) [inline .gitignore:3]: `*.sw[pon]` would be shorter

File: README.md [hunk 2]
```diff
@@ -1,64 +1,62 @@
...
+[Info]: <> (
+	This is the list of all the files that end with .example
+)
+Once the repository is cloned you can remove the `.example` at the end of the following files:
+- login/.env.example
+- Interface/src/contexts/SampleContext.example.tsx
+- esp32/config/secrets.example.yaml
+
+---
+Once done you want to get the ip address of the computer that will run the docker containers
```
Comment (jpilet) [inline README.md:30]: Is IP address mandatory or can we use a host name?

File: README.md [hunk 3]
```diff
@@ -1,64 +1,62 @@
...
+[Info]: <> (
+	This is the list of all the files that require to know the jwt secret key
+)
+You then want to copy the secret key and paste it the following files:
+- login/.env
+- docker-compose.yml
+
+---
+Finally you can fill the `esp32/config/secrets.yaml` with the credentials of the wifi\
+Once that's done you can create a postgres database called `memoires-info` from the `psql.sql` file
```
Comment (jpilet) [inline README.md:59]: Db creation should be done with the POSTGRES_DB en Var. See doc https://hub.docker.com/_/postgres
See also the "Initialization scripts" section to pick psql.sql on the first execution.

File: README.md (hunk 2 above)
Comment (Rignchen) [reply README.md]: I've changed it and it now use a host name

File: .gitignore (hunk 1 above)
Comment (Rignchen) [reply .gitignore:3]: I didn't knew you could do that, thank's

**Pull request merged**